code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -244,7 +244,7 @@ emp_act
 	apply_damage(I.force, I.damtype, affecting, armor, sharp=weapon_sharp, edge=weapon_edge, used_weapon=I)
 
 	var/bloody = 0
-	if(((I.damtype == BRUTE) || (I.damtype == HALLOSS)) && prob(25 + (I.force * 2)))
+	if(I.damtype == BRUTE)

```

Comment:
Yeah, I knew I'd misread that--I was so focused on the arguments and such that ..yeah.